agent/containers/images/gen-tags-from-rpm

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Generate 3 lines of output to stdout containing strings which
+# can be used as image tags.
+#
+# The first line emitted contains one tag representing the Git
+# SHA1 short hash label taken from the RPM's commit ID suffix.
+#
+# The second line contains the full version string without the
+# Git commit ID suffix.
+#
+# The third line contains two separate labels separate by
+# white space, the first being the "latest" label using only
+# the version's "major" ID, the second being the "latest" label
+# using the versions' "major" and "minor" ID.
```

Comment:
"version's" ... although I think the use of "version's" here and in the previous clause are probably not particularly useful anyway since "major ID" and "minor ID" are fairly clear concepts... and use of the possessive for an inanimate object is always grammatically awkward. ;-)